src/moscot/problems/base/_birth_death.py

```diff
@@ -68,7 +68,7 @@ def score_genes_for_marginals(
         to proliferation and/or apoptosis must be passed.
 
         Alternatively, proliferation and apoptosis genes for humans and mice are saved in :mod:`moscot`.
-        The gene scores will be used in :meth:`moscot.problems.TemporalProblem.prepare` to estimate the initial
+        The gene scores will be used in :meth:`~moscot.problems.TemporalProblem.prepare` to estimate the initial
```

Comment:
Just a question: is this mixin class going to always be used with `TemporalProblem`? If not, would consider referring to a more generic `prepare`.